flathub.json

```diff
@@ -0,0 +1,4 @@
+{
+    "skip-appstream-check": true,
```

Comment:
```suggestion
```
Appdata files must be validated

Reply:
Sorry, saw that in a couple of elementary apps on flathub and copied.